Binding quickstart - Cron input PostgreSQL output quickstart doc (#2471)
dapr/docs: daprdocs/content/en/getting-started/quickstarts/_index.md

| [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) |  Call a method on another service using the service invocation API. |
| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Send and receive messages using the publish and subscribe API. |
| [State Management]({{< ref statemanagement-quickstart.md >}}) | Create stateful applications using the state management API. |

> | [Bindings]({{< ref bindings-quickstart.md >}}) | Use Dapr's input and output bindings to schedule a database insert job. |

Reviewer: ```suggestion
| [Bindings]({{< ref bindings-quickstart.md >}}) | Schedule a database insert job using the input and output bindings API. |
```

To match the sentence structure of the other quickstart descriptions.
